### PR TITLE
Fix 18576

### PR DESCRIPTION
**Pull request by thibaultzanini**
- [x] table needs styling
- [x] table needs graphic state (searching, no data found).

### CODE REVIEW

**Comment by thibaultzanini** [comment]
@joshuarule FYI

File: ui/sections/containers/controls/docker-image-search.reel/docker-image-search.html [hunk 1]
```diff
@@ -43,6 +56,10 @@
     <div data-montage-id="owner" class="DockerImageSearch">
         <div data-montage-id="collection"></div>
         <div data-montage-id="image"></div>
+        <div data-montage-id="searching"></div>
     </div>
 </body>
 </html>
+
+<!-- ,
```
Comment (pchaussalet) [inline ui/sections/containers/controls/docker-image-search.reel/docker-image-search.html]: It should probably not have been added. ;)

File: ui/controls/table.reel/_table.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.Table { border: 1px solid #16171A; }
+/* .Table { border: 1px solid #16171A; } */
```
Comment (pchaussalet) [inline ui/controls/table.reel/_table.css]: It should be removed instead of commented.

File: ui/controls/table.reel/_table.css [hunk 3]
```diff
@@ -25,21 +25,21 @@
 
 }
 
-.Table-label {
+/* .Table-label {
```
Comment (pchaussalet) [inline ui/controls/table.reel/_table.css]: Same here

**Review by pchaussalet — CHANGES_REQUESTED** [review]

**Review by pchaussalet — APPROVED** [review]
LGTM

**Pull request merged**